components/auth/authProvider.js

```diff
@@ -17,14 +16,14 @@ module.exports = () => {
 		}),
 		);
 		app.use(bodyParser.urlencoded({ extended: true }));
-		app.use(cors());
```

Comment:
Why cors is being removed?

Reply:
Understandable concern, I didn't see the need for this any longer, it has no affect on the login process of the fetching of the namespaces where we extract the service bus messages. 

I have created a new work item on this project to resolves all of the routing issue we have in the bus inspector and I don't think we should rely on a 3rd part package to do our routing and request handling for us.